source/fundamentals/data-formats/document-data-format-pojo.txt

```diff
@@ -13,17 +13,215 @@ Document Data Format: POJOs
 
-- Description; convert between BSON and POJOs  
-- Summary of contents of this doc
+This guide explains how you can store and retrieve MongoDB data using plain
+old Java objects (`POJOs <https://en.wikipedia.org/wiki/Plain_old_Java_object>`__).
+POJOs are classes that do not depend on any other files or frameworks, and are
+often designed for storage and access of data.
 
-CodecRegistry
--------------
+.. _fundamentals-example-pojo:
 
-- Brief introduction, link to data-formats/codecs for more info
-- Example demonstrating the following:
-  - specifying how to create a CodecRegistry that uses PojoCodecProvider
-  - example POJO class with getters/setters
-  - include user-defined classes and lists
-- Sample input/output of aforementioned example
+Example POJO
+------------
 
-- Link to data-formats/pojo-customization for more information
+To follow the steps in this guide, you can create your own class or use the
+following sample POJO class which describes characteristics of a flower:
+
+.. code-block:: java
+
+   public class Flower {
+
+       private ObjectId id;
+       private String name;
+       private List<String> colors;
+       private Boolean isBlooming;
+       private Float height;
+
+       // public empty constructor needed for retrieving the POJO
+       public Flower() {}
+
+       public Flower(String name, Boolean isBlooming, Float height, List<String> colors) {
+           this.name = name;
+           this.isBlooming = isBlooming;
+           this.height = height;
+           this.colors = colors;
+       }
+
+       public ObjectId getId() {
+           return id;
+       }
+
+       public void setId(ObjectId id) {
+           this.id = id;
+       }
+
+       public String getName() {
+           return name;
+       }
+
+       public void setName(String name) {
+           this.name = name;
+       }
+
+       public Boolean getIsBlooming() {
+           return isBlooming;
+       }
+
+       public void setIsBlooming(Boolean isBlooming) {
+           this.isBlooming = isBlooming;
+       }
+
+       public Float getHeight() {
+           return height;
+       }
+
+       public void setHeight(Float height) {
+           this.height = height;
+       }
+
+       public List<String> getColors() {
+           return colors;
+       }
+
+       public void setColors(List<String> colors) {
+           this.colors = colors;
+       }
+
+       @Override
+       public String toString() {
+           return "Flower [id=" + id + ", name=" + name + ", colors=" + colors + ", isBlooming=" + isBlooming + ", height=" + height + "]";
+       }
+   }
+
+.. note:: POJO Serialization
+
+   Make sure your POJO properties meet the following requirements for
+   serialization:
+
+   - **public** getter methods
+   - **public** setter methods
+   - **public** fields
+
+   Also note that:
+
+   - The codec omits static and transient fields in your POJO.
+   - If your field has a corresponding getter method, the codec calls it for
```

Comment:
Suggestion: It might be useful to define what determines a "corresponding getter method". I'd assume that this refers to a method with:
- the name [get|set]<fieldName>
- parameter/return types that match the field's type
I know that some folks have different conventions for accessors and mutators, so we might want to clarify our assumption about what "getter" and "setter" mean.